core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -671,6 +671,18 @@ class CivilizationInfo : IsPartOfGameInfoSerialization {
         return diplomacyManager.hasOpenBorders
     }
 
+    fun getEnemyMovementPenalty(enemyUnit: MapUnit): Float {
+        val greatWallUnique = getMatchingUniques(UniqueType.EnemyLandUnitsSpendExtraMovement)
```

Comment:
Don't forget to include the `stateForConditionals` as the second parameter, otherwise conditionals cannot be used for this unique